src/moveFile.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFileContentAndMove(source, destination) {
+  try {
+    const fileContent = fs.readFileSync(source, (err) => {
+      if (err) {
+        return err;
+      }
+    });
+
+    fs.writeFileSync(destination, fileContent, (err) => {
```

Comment:
Use fs.writeFileSync without a callback, as it is synchronous.

Reply:
Done